tests/x11/tracker/tracker_by_command.pm

```diff
@@ -24,19 +24,20 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_leap);
 
 sub run {
     x11_start_program('xterm');
     if (is_sle('<12-SP2')) {
         script_run "tracker-search newfile";
     }
     else {
-        script_run 'tracker search emtpyfile';
+        my $trackercmd = (is_sle('<16') or is_leap('<16.0')) ? 'tracker' : 'tracker3';
+        script_run "$trackercmd search emtpyfile";
```

Comment:
As you touch it anyway,

```suggestion
        script_run "$trackercmd search emptyfile";
```

Reply:
mwahaha - that might well be the reason for the so-called 'softfail' - prep touches emptyfile already, but search looks for a different name. Good catch

Reply:
Apparently it softfails in 100% of runs, so if a verification run is green, you can remove that and the code for tracker3 can be simpler

Reply:
the test is that it 'searches an inexisting file' (-> nor result) and then makes a screen check if there was no result found; as the screen UI sufficiently changed, no needle matches the 'no file found' and thus it believes a file was found.

assert_screen would be better imho:

assert_screen('tracker-empty-file'); then we can create a needle with a match on found, and a needle with softfail flag if not found